src/app/scripts/cac/map/cac-map-templates.js

```diff
@@ -258,36 +267,41 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
             '</div>',
             '{{#each data.legs}}',
                 '<div class="directions-leg" ',
-                    'data-lat="{{this.from.lat}}" data-lon="{{this.from.lon}}">',
-                    // transit step directions
+                'data-lat="{{this.from.lat}}" data-lon="{{this.from.lon}}">',
                     '{{#if this.transitLeg}}',
-                    '<div class="directions-step {{modeClass this.mode}}" ',
-                        'data-lat="{{ this.from.lat }}" data-lon="{{ this.from.lon }}">',
-                        '<div class="directions-instruction">Board {{this.agencyName}} {{this.route}} ',
-                        '{{this.headsign}}</div>',
-                        '<div class="directions-time">at {{datetime this.startTime}}</div>',
-                        '<div class="directions-distance">{{inMiles this.distance}} mi</div>',
-                    '</div>',
-                    '<div class="directions-step directions-step-disembark" ',
-                        'data-lat="{{ this.to.lat }}" data-lon="{{ this.to.lon }}">',
-                        '<div class="directions-instruction">Disembark <strong>',
-                            '{{this.to.name}}</strong></div>',
-                    '</div>',
-                    '{{else}}',
-                    // non-tranist step directions
-                    '{{#each steps}}',
-                        '<div class="directions-step {{directionClass this.relativeDirection ../this.mode}}" ',
-                            'data-lat="{{ lat }}" data-lon="{{ lon }}">',
-                            '<div class="directions-instruction">{{directionText}}</div>',
+                        // transit step directions
```

Comment:
It's unclear to me why the transit step directions here have been moved outside of the `step` block of the Handlebars template. References to `this` here will now refer to the `leg` instead of the `step`. It seems that it's fine though in practice, and [here](https://github.com/opentripplanner/OpenTripPlanner/blob/f60440f12f9465390bc39dea7ee8e120d382eb3f/src/main/java/org/opentripplanner/api/model/Leg.java#L222) it looks like OTP will only return a single step for transit.

My concern is that the info may be different for the leg, like departure/arrival times not matching... seems fine though.